packages/dcos-integration-test/extra/test_helpers.py

```diff
@@ -1,3 +1,4 @@
 from pkgpanda.util import load_json
 
 dcos_config = load_json('/opt/mesosphere/etc/expanded.config.json')
+expanded_config = load_json('/opt/mesosphere/etc/expanded.config.json')
```

Comment:
we had that in EE. This is open so we had to add it. In theory I should have removed dcos_config as well, but I wanted to really minimize the changes on this branch